styles/main.css

```diff
@@ -3,6 +3,11 @@ body {
   width: 389px;
 }
 
+.container {
+  background-color: rgb(248, 246, 246);
+  width: 389px;
```

Comment:
- [ Optional ] Using fixed width makes your website not responsive. So, please remove fixed sized widths in any of your html elements.

Reply:
@gtekle , This is noted. Please provide me with alternative to fixed width since removing the elements distorts my layout.

Also this being optional, I hope the next reviewer wont be harsh on the same :smiley:

Reply:
Hey 

> @gtekle , This is noted. Please provide me with alternative to fixed width since removing the elements distorts my layout.
> 
> Also this being optional, I hope the next reviewer won't be harsh on the same smiley

Hey @kiboma2021,
Alternatives you can use instead of fixed-width are `vw` or `%`. For example, `width: 100%;` or `width: 100vw;` 

I'm sorry that you feel that the previous reviewers were harsh. The required changes are meant to make sure that you learn as much as possible in order to become a professional developer. I will review your changes shortly.